packages/definitions/dist/fhir/r4/profiles-medplum.json

```diff
@@ -482,6 +482,16 @@
               "code" : "string"
             }]
           },
+          {
+            "id" : "Bot.runtimeVersion",
+            "path" : "Bot.runtimeVersion",
+            "definition" : "The identifier of the bot runtime environment (i.e., vmcontext, awslambda, etc).",
```

Comment:
For clarity in logging, etc. is there a good reason not to make the runtime string something more human readable? Like `vmcontext-0.0` or `awslambda-0.1`?

Reply:
Yeah, I think it's worth thinking about a better long term strategy.  

We may want to include AWS's own system as part of this code, or maybe an additional field: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

For now, I propose we stick with something simple.